webhooks.md

```diff
@@ -114,4 +116,82 @@ When an action of particular type is performed in the system, a message is sent
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
 | `DeletedUtc` | string | required | Deletion date and time of the integration in UTC timezone in ISO 8601 format. |
-| `Integration` | [Integration](#integration) | required | Integration data. |
\ No newline at end of file
+| `Integration` | [Integration](#integration) | required | Integration data. |
+
+## General message
+
+This is a new format of webhook message which will be extended in future to carry events about different entities in Mews.
+
+```javascript
+{
+    "EnterpriseId": "851df8c8-90f2-4c4a-8e01-a4fc46b25178",
+    "IntegrationId": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+    "Events": [
+        {
+            "Discriminator": "ServiceOrderUpdated",
+            "Value": {
+                "Id": "bfee2c44-1f84-4326-a862-5289598f6e2d"
+            }
+        }
+    ],
+    "Entities": {
+        "ServiceOrders": [
+            {
+                "Id": "bfee2c44-1f84-4326-a862-5289598f6e2d",
+                "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+                "GroupId": "94843f6f-3be3-481b-a1c7-06458774c3df",
+                "Number": "52",
+                "State": "Processed",
+                "Origin": "Connector",
+                "CreatedUtc": "2016-02-20T14:58:02Z",
+                "UpdatedUtc": "2016-02-20T14:58:02Z",
+                "CancelledUtc": null,
+                "StartUtc": "2016-02-20T13:00:00Z",
+                "EndUtc": "2016-02-22T11:00:00Z",
+                "RequestedCategoryId": "773d5e42-de1e-43a0-9ce6-f940faf2303f",
+                "AssignedResourceId": "20e00c32-d561-4008-8609-82d8aa525714",
+                "AssignedResourceLocked": false,
+                "BusinessSegmentId": null,
+                "CompanyId": null,
+                "TravelAgencyId": null,
+                "AvailabilityBlockId": null,
+                "RateId": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+                "AdultCount": 2,
+                "ChildCount": 0,
+                "CustomerId": "35d4b117-4e60-44a3-9580-c582117eff98",
+                "VoucherId": null
+            }
+        ]
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `EnterpriseId` | string | required | Unique identifier of [Enterprise](configuration.md#enterprise) where events belong to. |
+| `IntegrationId` | string | required | Unique identifier of [Integration](#integration) which events are connected to. |
+| `Events` | array of [Event](#event) | required | List of events that happened in Mews since last webhook call. |
```

Comment:
nit: Maybe I wouldn't define it that specifically, webhooks might even be going in parallel in future, so even though it works like this internally at the moment, I wouldn't mention it here. So something like `The events that occurred in Mews.`.